source/fundamentals.txt

```diff
@@ -18,3 +18,4 @@ Fundamentals
    /fundamentals/authentication
    /fundamentals/enterprise-authentication
    /fundamentals/linq
+   /fundamentals/class-mapping
```

Comment:
Good idea on the cross linking, added! 

I'm not sure if data formats would be the best place for this page, since it's more about moving the data back and forth rather than the format the data is in, even though it is using POCOs. I also think this is probably used widely enough to where it would benefit from not being nested on the toc tree. Happy to discuss more if you'd like!